test/support/application_env_helpers.ex

```diff
@@ -0,0 +1,36 @@
+defmodule RingLogger.ApplicationEnvHelpers do
+  @doc """
+  Set application env for one test, then resets it after
+
+  Add the following tag to set the configuration for `:ring_logger`:
+
+  ```
+  @tag application_envs: [ring_logger: [colors: %{debug: :green, error: :blue}]]
+  ```
+  """
+  def with_application_env(%{application_envs: application_envs} = context, on_exit)
```

Comment:
This is really cool. I'm going to steal this for other projects.

Reply:
Yeah, go ahead! I want to get it into some sort of shared ex_unit library, but I'm not sure what the correct one would be. There's `assertions` but it's more focused on assertions than on test helpers.